docs/source/processors/panel.rst

```diff
@@ -17,10 +20,15 @@ Required Tag Parameters
 
 - ``title`` - Text to display as the panel's title.
 
+    - This is not given as a parameter within the ``{panel}`` tag, but rather as the first line of the panel with a Level 1 Heading.
+
 Optional Tag Parameters
 ***************************************
 
 - ``subtitle`` - Text to display as the panel's subtitle after the title.
```

Comment:
As above, this doesn't really describe what the parameter is anymore

Perhaps consider something similar to the following
```subtitle - Boolean flag to indicate whether the panel should display a subtitle. If set to "true", then the subtitle is the first Level 2 Heading within the panel.```